src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,102 @@
+/*  eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import classnames from 'classnames'
+
+import { Address } from './Address/Address'
+import { Button } from '../Button/Button'
+import { Footer } from './Footer'
+import { Logo } from './Logo/Logo'
+
+export default {
+  title: 'Footer',
+  parameters: {
+    info: `
+  USWDS 2.0 Footer component
+  
+  Source: https://designsystem.digital.gov/components/form-controls/#footer
+  `,
+  },
+}
+
+type SizeProps = {
+  big?: boolean
+  medium?: boolean
+  slim?: boolean
+}
+
+// Placeholder until a dynamic Nav is built out and can be used
+const MockNav = ({ medium, slim }: SizeProps): React.ReactElement => {
+  const itemClasses = classnames(
+    'desktop:grid-col-auto usa-footer__primary-content',
+    {
+      'mobile-lg:grid-col-4': medium,
+      'mobile-lg:grid-col-6': slim,
+    }
+  )
+
+  const items = Array(4).fill({
+    href: 'javascript:void(0);',
+    heading: 'Primary Link',
+  })
+
+  return (
+    <nav className="usa-footer__nav" aria-label="Footer navigation">
+      <ul className="grid-row grid-gap">
+        {items.map((item, i) => (
+          <li key={`navItem-${i}`} className={itemClasses}>
+            <a className="usa-footer__primary-link" href={item.href}>
+              {item.heading}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </nav>
+  )
+}
+
+const returnToTop = (
```

Comment:
Do you think this should be made a subcomponent as well? it could take an onClick handler (and i18n text)